src/plugin.js

```diff
@@ -226,6 +228,29 @@ const onPlayerReady = (player, emeError) => {
       setupSessions(player);
       handleWebKitNeedKeyEvent(event, getOptions(player), player.tech_)
         .catch(emeError);
+    };
+
+    // Support Safari EME with FairPlay
+    // (also used in early Chrome or Chrome with EME disabled flag)
+    player.tech_.el_.addEventListener('webkitneedkey', (event) => {
+      // It's possible that at the start of playback a rendition switch
+      // on a small player in safari's HLS implementation will cause
```

Comment:
Does it have to be a small player?

Reply:
makes it more likely since they'll start with a lower rendition. Happens a lot less if you have the window maximized.